### ISSUE TITLE
Generic Template Issue: 501 #5.5.4 Cannot decode AUTH parameter

### ISSUE THREAD

**Issue by vidaria** [pentestgeek/phishing-frenzy#366]
I have setup my pf in an ubuntu server 16.04.2 LTS version and get it up and near ready to run but I  got stuck in this block as when i try send  a test message to myself it gives me this error.   have use the default template like efax and intel password still the same also clone the template from the pf website and no change at all even  i just create an easy plain no link templates for the testing but still giving me the same error as showed below.  apached and redis and sidekiq are all green in the web browser and I am not sure what to do.

Generic Template Issue: 501 #5.5.4 Cannot decode AUTH parameter

I am not really sure what to do with it but i really need help pls.

**Comment by zeknox** [comment]
Sounds like an issue with your SMTP configurations. The error looks to becoming from the SMTP server your trying to send through.

Although it seems strange this would be coming from within a generic template issue.

Try some different smtp settings and report back.

Brandon

**Comment by vidaria** [comment]
Yes Brandon you are right on that it was my smtp setting tap.  I did choose it to plain but then I went back and check and change to none and test mail send and campaign launched too.  
It was seem a little to trick to me when it said generic templates error and i thought at first it was something to do with my templates but it was fix by changing my smtp authenticate tap to none.

Thanks heap for the help it much appreciated and the tools are awesome.  keep up  the good work.
